# This file is auto-generated by the dh-umc tools and should not be edited!
msgid ""
msgstr ""
"Project-Id-Version: univention-management-console-module-adtakeover\n"
"Report-Msgid-Bugs-To: packages@univention.de\n"
"POT-Creation-Date: 2016-08-18 16:57+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: \n"

#: umc/python/adtakeover/takeover.py:1960
msgid " failed"
msgstr ""

#: umc/python/adtakeover/takeover.py:812
#, python-format
msgid "AD Server did not return proper time string: %s."
msgstr ""

#: umc/python/adtakeover/takeover.py:619
#, python-format
msgid "Active Directory services not detected at %s."
msgstr ""

#: umc/python/adtakeover/takeover.py:1485
msgid "Active Directory takeover finished already."
msgstr ""

#: umc/python/adtakeover/takeover.py:327
msgid "Adjusting settings in Samba directory service"
msgstr ""

#: umc/python/adtakeover/takeover.py:203
msgid ""
"At least one GPO in SYSVOL is newer than the Group Policy Container version."
msgstr ""

#: umc/python/adtakeover/takeover.py:200
#: umc/python/adtakeover/takeover.py:1887
msgid "At least one GPO in SYSVOL is not up to date yet."
msgstr ""

#: umc/python/adtakeover/takeover.py:197
msgid "At least one GPO is still missing in SYSVOL."
msgstr ""

#: umc/python/adtakeover/takeover.py:243 umc/python/adtakeover/takeover.py:270
msgid "Authenticating"
msgstr ""

#: umc/python/adtakeover/takeover.py:191
msgid "Authentication failed."
msgstr ""

#: umc/python/adtakeover/takeover.py:382
msgid "Checking GPOs in SYSVOL"
msgstr ""

#: umc/python/adtakeover/takeover.py:291
msgid "Checking group policies"
msgstr ""

#: umc/python/adtakeover/takeover.py:331
msgid "Claiming FSMO roles"
msgstr ""

#: umc/python/adtakeover/takeover.py:240 umc/python/adtakeover/takeover.py:266
#, python-format
msgid "Connecting to %s"
msgstr ""

#: umc/python/adtakeover/takeover.py:1996
#, python-format
msgid "Connection to Active Directory server %s failed."
msgstr ""

#: umc/python/adtakeover/takeover.py:927 umc/python/adtakeover/takeover.py:937
#, python-format
msgid "Copying %s"
msgstr ""

#: umc/python/adtakeover/takeover.py:909
#, python-format
msgid "Could not read local machine password: %s"
msgstr ""

#: umc/python/adtakeover/takeover.py:194
msgid "Domain join failed."
msgstr ""

#: umc/python/adtakeover/__init__.py:69
#, python-format
msgid ""
"Execution of command '%(command)s' has failed:\n"
"\n"
"%(text)s"
msgstr ""

#: umc/python/adtakeover/takeover.py:2029
msgid "Faild to detect the Active Directory site of the server."
msgstr ""

#: umc/python/adtakeover/takeover.py:671
msgid "Failed to determine AD domain SID."
msgstr ""

#: umc/python/adtakeover/takeover.py:1016
msgid "Failed to determine new domain SID."
msgstr ""

#: umc/python/adtakeover/takeover.py:353
msgid "Finalizing"
msgstr ""

#: umc/python/adtakeover/takeover.py:293
msgid "Initializing the S4 Connector listener"
msgstr ""

#: umc/python/adtakeover/takeover.py:218
msgid "Insufficient License."
msgstr ""

#: umc/python/adtakeover/takeover.py:566
msgid "Internal Error: License check failed."
msgstr ""

#: umc/python/adtakeover/takeover.py:833
#, python-format
msgid "Internal Error: rdate -s -p failed (%d)."
msgstr ""

#: umc/python/adtakeover/takeover.py:426
#, python-format
msgid ""
"Internal module error: Cannot go from state '%(current)s' to state "
"'%(new)s'."
msgstr ""

#: umc/python/adtakeover/takeover.py:431
#, python-format
msgid ""
"Internal module error: Expected to be in state 'sysvol', but found '%s'."
msgstr ""

#: umc/python/adtakeover/takeover.py:436
#, python-format
msgid ""
"Internal module error: Expected to be in state 'takeover', but found '%s'."
msgstr ""

#: umc/python/adtakeover/takeover.py:409
#, python-format
msgid "Internal module error: Refusing to set invalid state '%s'."
msgstr ""

#: umc/python/adtakeover/takeover.py:441
msgid "Internal module error: Takeover running, aborting attempt to restart."
msgstr ""

#: umc/python/adtakeover/takeover.py:2472
#, python-format
msgid "Invalid UNC string: %s"
msgstr ""

#: umc/python/adtakeover/takeover.py:468
#, python-format
msgid "Invalid state in file %s"
msgstr ""

#: umc/python/adtakeover/takeover.py:279
msgid "Joining the domain"
msgstr ""

#: umc/python/adtakeover/takeover.py:1963
#, python-format
msgid "Network connection to %s failed."
msgstr ""

#: umc/python/adtakeover/takeover.py:590
#, python-format
msgid ""
"Number of %(object_name)s after takeover would be %(sum)s. This would exceed"
" the number of licensed objects (%(max)s)."
msgstr ""

#: umc/python/adtakeover/takeover.py:301
msgid "Rebuilding IDMAP"
msgstr ""

#: umc/python/adtakeover/takeover.py:350
msgid "Reconfiguring nameserver"
msgstr ""

#: umc/python/adtakeover/takeover.py:347
msgid "Registering IP in DNS"
msgstr ""

#: umc/python/adtakeover/takeover.py:822
#, python-format
msgid ""
"Remote clock is behind local clock by more than %s seconds, refusing to turn"
" back time. Please advance the clock of the Active Directory DC."
msgstr ""

#: umc/python/adtakeover/takeover.py:334
msgid "Removing the previous AD server account"
msgstr ""

#: umc/python/adtakeover/takeover.py:304
msgid "Reset SYSVOL ACLs"
msgstr ""

#: umc/python/adtakeover/takeover.py:246
msgid "Retrieving information from AD DC"
msgstr ""

#: umc/python/adtakeover/takeover.py:288
msgid "Rewriting SIDs in the UCS directory service"
msgstr ""

#: umc/python/adtakeover/takeover.py:321
#, python-format
msgid "Search for %s in network"
msgstr ""

#: umc/python/adtakeover/takeover.py:206
msgid "Something is wrong with the SYSVOL."
msgstr ""

#: umc/python/adtakeover/takeover.py:285
msgid "Starting Samba"
msgstr ""

#: umc/python/adtakeover/takeover.py:297
msgid "Starting the S4 Connector"
msgstr ""

#: umc/python/adtakeover/takeover.py:274
msgid "Synchronizing system clock"
msgstr ""

#: umc/python/adtakeover/takeover.py:326
msgid "Taking over Active Directory domaincontroller roles"
msgstr ""

#: umc/python/adtakeover/takeover.py:338
msgid "Taking over DNS address"
msgstr ""

#: umc/python/adtakeover/takeover.py:344
msgid "Taking over IP address"
msgstr ""

#: umc/python/adtakeover/takeover.py:341
msgid "Taking over NETBIOS address"
msgstr ""

#: umc/python/adtakeover/takeover.py:667
#, python-format
msgid ""
"The Active Directory domain has a function level of Windows Server 2012 or "
"newer, Samba currently only supports up to Windows 2008R2: %s"
msgstr ""

#: umc/python/adtakeover/takeover.py:1458
#: umc/python/adtakeover/takeover.py:1466
msgid "The Active Directory domain join was not completed successfully yet."
msgstr ""

#: umc/python/adtakeover/takeover.py:209
msgid "The Active Directory server seems to be running. It must be shut off."
msgstr ""

#: umc/python/adtakeover/takeover.py:652
msgid ""
"The LDAP base of this UCS domain differs from the LDAP base of the selected "
"Active Directory domain."
msgstr ""

#: umc/python/adtakeover/takeover.py:1510
#, python-format
msgid "The Server IP %s is still online, please shut down the machine."
msgstr ""

#: umc/python/adtakeover/takeover.py:2005
#, python-format
msgid ""
"The automatic search for an Active Directory server for realm %s did not "
"yield any results."
msgstr ""

#: umc/python/adtakeover/takeover.py:188
msgid "The computer is not reachable."
msgstr ""

#: umc/python/adtakeover/takeover.py:858 umc/python/adtakeover/takeover.py:948
#: umc/python/adtakeover/takeover.py:996
#, python-format
msgid "The domain join failed. See %s for details."
msgstr ""

#: umc/python/adtakeover/takeover.py:647
msgid ""
"The selected Active Directory server has the same NTDS GUID as this UCS "
"server."
msgstr ""

#: umc/python/adtakeover/takeover.py:215
msgid "Time difference critical for Kerberos but synchronization aborted."
msgstr ""

#: umc/python/adtakeover/takeover.py:212
msgid "Time synchronization failed."
msgstr ""

#: umc/python/adtakeover/takeover.py:2465
msgid "UNC doesn't start with \\\\ or //"
msgstr ""

#: umc/python/adtakeover/takeover.py:251
msgid ""
"You may proceed with the takeover, but the domain management may be limited "
"afterwards until a new license is installed."
msgstr ""

#: umc/python/adtakeover/takeover.py:577
msgid "users"
msgstr ""
